app/presenters/summary/html_presenter.rb

```diff
@@ -19,8 +19,9 @@ def sections
         Sections::JustificationForLegalAid.new(crime_application),
         Sections::PassportJustificationForLegalAid.new(crime_application),
         Sections::LegalRepresentativeDetails.new(crime_application),
+        Sections::SupportingEvidence.new(crime_application),
```

Comment:
Although for now it doesn't matter as it is feature-flagged, I think this section will go before the Legal rep details (which is the Declaration and thus the last step).

Reply:
Oh ok, will move it